setup.py

```diff
@@ -50,7 +50,7 @@
     zip_safe=False,
 
     install_requires=[
-        'ftw.simplelayout [contenttypes] >= 1.12.0',
+        'ftw.simplelayout [contenttypes] > 1.21.1',
```

Comment:
I'd change that to `>=1.22.0` or whatever, after `ftw.simplelayout` is released. It's more explicit 😉